### ISSUE TITLE
Panic recovery -> invalid character '<' looking for beginning of value

### ISSUE THREAD

**Issue by jarredkenny** [etcinit/phabulous#43]
Hello,

I am using the latest release (2.3.0) and have configured the bot using the main/config.yml file. When I run the bot, it seems to be working and it says `Hi! Phabulous v2 reporting for duty` in the channel I have it in. 

However, when feed events occur within my Phabricator installation Phabulous logs the following.

```
2016/10/05 12:59:32 Panic recovery -> invalid character '<' looking for beginning of value
/usr/local/Cellar/go/1.5.1/libexec/src/runtime/panic.go:423 (0x42abc9)
/Users/etcinit/go/src/github.com/etcinit/phabulous/app/controllers/feed.go:37 (0x5433b8)
/Users/etcinit/go/src/github.com/etcinit/phabulous/app/controllers/feed.go:29 (0x544bca)
/Users/etcinit/go/src/github.com/gin-gonic/gin/context.go:95 (0x54509a)
/Users/etcinit/go/src/github.com/gin-gonic/gin/logger.go:56 (0x556541)
/Users/etcinit/go/src/github.com/gin-gonic/gin/context.go:95 (0x54509a)
/Users/etcinit/go/src/github.com/gin-gonic/gin/recovery.go:43 (0x5570a1)
/Users/etcinit/go/src/github.com/gin-gonic/gin/context.go:95 (0x54509a)
/Users/etcinit/go/src/github.com/gin-gonic/gin/gin.go:294 (0x54baf2)
/Users/etcinit/go/src/github.com/gin-gonic/gin/gin.go:275 (0x54b727)
/usr/local/Cellar/go/1.5.1/libexec/src/net/http/server.go:1862 (0x5fe5fe)
/usr/local/Cellar/go/1.5.1/libexec/src/net/http/server.go:1361 (0x5fbc3e)
/usr/local/Cellar/go/1.5.1/libexec/src/runtime/asm_amd64.s:1696 (0x45c141)
```

Any idea what might be happening?

**Comment by mygon0172** [comment]
I also encountered this problem

**Comment by schemar** [comment]
This should be fixed in the upcoming 3.0.0 release.
